lightning/src/ln/channelmanager.rs

```diff
@@ -10478,7 +10478,7 @@ where
 	/// # Custom Routing Parameters
 	///
 	/// Users can customize routing parameters via [`RouteParametersConfig`].
-	/// To use default settings, call the function with [`RouteParametersConfig::default()`].
+	/// To use default settings, call the function with [`RouteParametersConfig::default`].
```

Comment:
nit: Perhaps this commit can be squashed with the previous docs update before merge?